npm-shrinkwrap.json

```diff
@@ -2,7 +2,7 @@
   "name": "marathon-ui",
   "version": "0.9.0",
   "npm-shrinkwrap-version": "5.4.0",
-  "node-version": "v0.10.38",
```

Comment:
You can tell Travis what version of Node you [want it to use](http://docs.travis-ci.com/user/languages/javascript-with-nodejs/).